### PR TITLE
New cookbook recipe reg. multisite setup

### PR DESCRIPTION
**Pull request by Adspectus**
I have created a new recipe which shows an alternative setup for multisites. Please review and feel free to comment.

### CODE REVIEW

File: content/1_docs/2_cookbook/9_setup/0_multisite-variant/recipe.txt [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+The symbolic links are created like this:
+
+```bash
+user@host:/srv/www/my-site.com $ ln -s /srv/www/kirby kirby
+user@host:/srv/www/my-shop.com $ ln -s /srv/www/kirby kirby
+```
+
+## In `index.php`
+
+php can follow the symlinked `kirby` directory like every else. However, since in `kirby/config/roots.php` the root to `index` is set up like this:
```
Comment (texnixe) [inline content/1_docs/2_cookbook/9_setup/0_multisite-variant/recipe.txt]: PHP not php

Comment (Adspectus) [reply content/1_docs/2_cookbook/9_setup/0_multisite-variant/recipe.txt]: Changed with fb99913a943c31d0efad68b73fc6e73e4d498cce

**Comment by bastianallgeier** [comment]
I really like it! Very clean and well-written. Wouldn't you want to include a little paragraph at the bottom about yourself though? Something like "This cookbook recipe was written by ..." with a link to your site or twitter profile or whatever you prefer?

**Comment by Adspectus** [comment]
> I really like it! Very clean and well-written. Wouldn't you want to include a little paragraph at the bottom about yourself though? Something like "This cookbook recipe was written by ..." with a link to your site or twitter profile or whatever you prefer?

Of course! But I could only link to my Kirby forum profile because I have no other so-called social media accounts...

**Comment by bastianallgeier** [comment]
@Adspectus that's fine for me!

**Comment by Adspectus** [comment]
> @Adspectus that's fine for me!

Did it.

**Review by bastianallgeier — APPROVED** [review]

**Comment by bastianallgeier** [comment]
Thank you so much for writing it! It inspired us to start a cookbook recipe program and to ask more external authors to contribute. Could you send me an email? I have a little surprise for you :) bastian@getkirby.com

**Pull request merged**